go.mod

```diff
@@ -4,7 +4,7 @@ go 1.18
 
 require (
 	github.com/CosmWasm/wasmd v0.22.0
-	github.com/cosmos/cosmos-sdk v0.45.4
+	github.com/cosmos/cosmos-sdk v0.45.9
```

Comment:
Is it okay to not update our SDK fork? We are using [SDK](https://github.com/regen-network/regen-ledger/pull/1549/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L151) forked version.